presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -296,6 +318,41 @@ public void testCacheRead(ReadMode readMode)
                     assertGreaterThan(getCachedReadsCount(), beforeCachedReadsCount);
                     assertEquals(getRemoteReadsCount(), firstRemoteReadsCount);
                 });
+
+        long secondRemoteReadsCount = getRemoteReadsCount();
```

Comment:
This test methods tends to look to long. Consider splitting into two (possibly calling out to shared private implementation).

Reply:
The test does:
1) writes test file
2) reads test file to warm-up cache and asserts counters (this ensures that cache is populated)
3) reads test file with cache enabled and asserts counters (this ensures that cache works)
4) reads test file with cache disabled via session property (this ensures that session property works)
5) reads test fail again with cache enabled using new FS (this ensures that cache data is not dropped) 

I guess maybe 3) could be removed, but I don't think it would greatly simplify this test.

Reply:
I was more thinking of having one test which does:
1,2,3. And other which is focused on session property and does 1,4,5. But not sure if that would be more readable.

Reply:
I'm not sure there isn't some interaction between 3) and 5), so that's why I prefer keep them together